template/scala-akka-http-server/api.mustache

```diff
@@ -1,14 +1,22 @@
 package {{package}}
 
+import akka.event.Logging
+import akka.http.scaladsl.model.HttpRequest
+import akka.http.scaladsl.server.directives.{DebuggingDirectives, LogEntry}
+import akka.http.scaladsl.server.{Directive0, Directive1, Route, RouteResult}
 import akka.http.scaladsl.server.Directives._
 import akka.http.scaladsl.model.StatusCodes
-import akka.http.scaladsl.server.{Directive1, Route}
+import org.slf4j.{Logger, LoggerFactory}
+import ch.qos.logback.classic.LoggerContext
+import it.pagopa.pdnd.interop.commons.logging.ContextFieldsToLog
```

Comment:
and should it?
AFAIK this template is shared with data lake team, and they do not have/need our commons library

Reply:
Actually it shouldn't, so probably the logging part may be either dropped or refactored without using `ContextFieldsToLog`.

Reply:
Good question. As far as I can tell (and I'm the last dev that worked on those), someone used this template to create the data lake microservices, but they have diverged slightly.
As per dropping or refactoring, I would aim for refactoring since it's already present in 90% of our microservices. I think that the whole operation can be a mere copy-pasting of the logging logic present in the dependency.

Reply:
fine by me.

After a [quick check of commons](https://github.com/pagopa/pdnd-interop-commons/blob/0.1.x/utils/src/main/scala/it/pagopa/pdnd/interop/commons/), it seems they are only:

1 - a type alias, namely `ContextFieldsToLog`
2 - two string constants, namely `UID` and `CORRELATION_ID_HEADER`.

so no actual logging logic from commons should be duplicated.

The only minor downside I see here is that if any of the constants shall change in the future, we should recall to update them properly both on commons and mustaches.